performance_curves/performance_curve_bundle.py

```diff
@@ -0,0 +1,54 @@
+from performance_curves.performance_curve import *
+
+
+class PerformanceCurveBundle:
+    def __init__(self,
+                 y_true: np.ndarray,
+                 y_scores: List[np.ndarray],
+                 metric: Metric,
+                 model_names: List[str],
+                 num_bins: Optional[int] = None,
+                 order_descending: bool = True,
+                 random_model: bool = True,
+                 random_seed: Optional[int] = None,
+                 num_trials: Optional[int] = 1,
+                 perfect_model: bool = True):
+        self.y_true = y_true
+        self.y_scores = y_scores
+        self.metric = metric
+        self.model_names = model_names
+        self.num_bins = num_bins
+
+        score_sizes = [len(y_score) for y_score in self.y_scores]
+        assert all(score_size == score_sizes[0] for score_size in score_sizes), \
+            "Number of predictions outputted by each model must be the same."
+
+        all_performance_values = []
+        all_case_counts = []
+        for i in range(len(self.y_scores)):
+            individual_curve = PerformanceCurve(self.y_true, y_scores[i], self.metric, self.num_bins,
+                                                order_descending)
+            all_performance_values.append(individual_curve.performance_values)
+            all_case_counts.append(individual_curve.case_counts)
+        self.bundle = dict(zip(self.model_names, zip(all_performance_values, all_case_counts)))
```

Comment:
Naming suggestion: This entire class is the bundle so this attribute should be called something else. Perhaps "curves"?

This way, the user's code won't read like `bundle.bundle["whaterver"]` which is confusing.

Reply:
Done!